watch_service.go

```diff
@@ -348,52 +322,53 @@ func (w *Watcher) doWatch() {
 
 	var lastKnownRevision int
 	curLatest := w.getLatest()
-	if curLatest == nil || curLatest.Revision == 0 {
-		lastKnownRevision = 1 // Init revision
+	if curLatest == nil || curLatest.Commit.Revision == 0 {
+		lastKnownRevision = -1 // Init revision
```

Comment:
We use the Init revision which is `1` to get the `latestRevision` of the file quickly.
Please see https://github.com/line/centraldogma/blob/3cfcc21a01e57ae16b5bb76ce1bb20412f8441c1/server/src/main/java/com/linecorp/centraldogma/server/internal/storage/repository/git/GitRepository.java#L1330 and
https://github.com/line/centraldogma/blob/30f439aec463f08193def834e72aa93b06d1eef1/client/java/src/main/java/com/linecorp/centraldogma/internal/client/AbstractWatcher.java#L224

Reply:
Ah I see, let me fix it. Thank you 🙇